app/controllers/problem_reports_controller.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+class ProblemReportsController < ApplicationController
+  def index
+    @problem_reports = ProblemReport.order(id: :desc)
+    filter_encode
+
+    if params[:filter] == 'Unfixed and Critical'
+      @problem_reports.where!(is_critical: true, is_fixed: false)
+    elsif params[:filter] == 'Unfixed and Fixable'
+      @problem_reports.where!(is_fixable: true, is_fixed: false)
+    end
+  end
+
+  def new; end
+
+  def create
+    create_new_report
+    if @report.save
+      flash[:notice] = 'Success'
+      redirect_to problem_reports_path
```

Comment:
You can `redirect_to problem_reports_path, notice: 'Success'`

Reply:
But you might want to give a better feedback to the user than just `Success`... :)